server/server.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+const connection = require('./database');
+
+app.get('/', function (req, res) {
+    let sql = "SELECT * FROM initial_broncohacks_db.User"
+    connection.query(sql, function (err, result) {
+        if (err) throw err;
+        res.send(result)
+    })
+});
+
```

Comment:
I tested it and your connection was set up to the DB correctly! Theres just a few things you need to change that I did not mention before (my fault gang).
- Following REST API practices, (https://restfulapi.net/), we need to send both a **status code** (like 200, 400, 500; more here https://restfulapi.net/http-status-codes/) and an optional **body containing  JSON**  (like how you already did for returnin all users)
- for the app.get('/'), it should be used to **test the connection to the DB** and not return all users.
  - What you should do instead is send a status code for if the connection was set up or not for now. You can add other stuff on top of it too such as the list of endpoints are use it for documentation but for now just send the status code + json like { message: "Succesful connection" } for status code 200 or like { message: "Failed to connect" } if it doesn't connect i forgot what status code it is
- If you wanna try out the get request for all users , do it in the Routes/user.js file and set up the router. Make sure to follow all of the aforementioned things above (status codes and json) when handling with cases such as user not found or a bad request. **Only do the 1 get request for retrieving all users**, as I want everyone to get the chance to make different endpoints